app/scripts/controllers/network/network.js

```diff
@@ -14,52 +14,41 @@ const {
   RINKEBY,
   KOVAN,
   MAINNET,
-  OLD_UI_NETWORK_TYPE,
-  DEFAULT_NETWORK,
+  LOCALHOST,
 } = require('./enums')
-const { getNetworkEndpoints } = require('./util')
+const LOCALHOST_RPC_URL = 'http://localhost:8545'
 const INFURA_PROVIDER_TYPES = [ROPSTEN, RINKEBY, KOVAN, MAINNET]
 
+const env = process.env.METAMASK_ENV
+const METAMASK_DEBUG = process.env.METAMASK_DEBUG
+const testMode = (METAMASK_DEBUG || env === 'test')
+
+const defaultProviderConfig = {
+  type: testMode ? RINKEBY : MAINNET,
+}
+
 module.exports = class NetworkController extends EventEmitter {
 
-  constructor (config) {
+  constructor (opts = {}) {
     super()
 
-    this._networkEndpointVersion = OLD_UI_NETWORK_TYPE
-    this._networkEndpoints = getNetworkEndpoints(OLD_UI_NETWORK_TYPE)
-    this._defaultRpc = this._networkEndpoints[DEFAULT_NETWORK]
-
-    config.provider.rpcTarget = this.getRpcAddressForType(config.provider.type, config.provider)
+    // parse options
+    const providerConfig = opts.provider || defaultProviderConfig
+    console.log('providerStore:', providerConfig)
```

Comment:
If this wasn't for debugging, should we use the `loglevel` module for consistency?

Reply:
oops, yeah this was for debugging